pages/deployment/wifi.md

```diff
@@ -104,7 +108,7 @@ To view an images current configuration run:
 $ sudo resin config read --type raspberry-pi
 ```
 
-If you'd like to reconfigure the image's network settings in an automated way you can use [`resin config write`](/pages/tools/resin-cli.md#config-write-60-key-62-60-value-62-).
+If you'd like to reconfigure the image's network settings in an automated way you can use [`resin config write`](/tools/resin-cli#config-write-60-key-62-60-value-62-).
```

Comment:
Do you know why the CLI generates such ugly links? /tools/resin-cli#config-write-60-key-62-60-value-62-